forge/test/mlir/llama/utils/utils.py

```diff
@@ -2,14 +2,14 @@
 
 # SPDX-License-Identifier: Apache-2.0
 
-from transformers import LlamaConfig, LlamaForCausalLM, LlamaTokenizer
+from transformers import LlamaConfig, LlamaForCausalLM, LlamaTokenizer, AutoTokenizer
```

Comment:
Can we use a specific tokenizer (e.g. Llama one)?

Reply:
Currently, only way to load llama 3.2 tokenizer is from AutoTokenizer